kythe/typescript/indexer.ts

```diff
@@ -1441,9 +1468,9 @@ class Visitor {
         if (funcSym && decl.parent.heritageClauses) {
           for (const heritage of decl.parent.heritageClauses) {
             for (const baseType of heritage.types) {
-              const baseSym =
-                  this.host.getSymbolAtLocation(baseType.expression);
-              if (!baseSym || !baseSym.members) {
+              const type =
+                  this.host.getTypeAtLocation(baseType.expression);
+              if (!type || !type.symbol || !type.symbol.members) {
```

Comment:
I don't understand this change, can you explain?

(Also, this function is getting extremely long, is there any reasonable way to refactor it a bit as you add this?)

Reply:
My understanding that for 

```ts
import {IFace} from './interface';

class Implementation implements IFace {}
```

When we call `getSymbolForLocation` it gives us location of the IFace symbol in the import statement, not the `IFace` symbol in `interface.ts`. But getting type works as type contains the main symbol from `interface.ts`. If there are more canonical ways to get main symbol of imported alias - I can switch to that.

Reply:
Ah interesting.  You can check if the symbol is an alias and then call .getAliasedSymbol() or whatever it's called, but this is fine I think.